xml/ay_bigfile.xml

```diff
@@ -2761,7 +2761,10 @@ openssl x509 -noout -fingerprint -sha256
 &lt;/subvolumes&gt;</screen>
         </entry>
         <entry>
-         <para/>
+         <para>
+          If no subvolumes section has been defined in a partition description,
```

Comment:
Maybe use `<tag>subvolumes</tag>` here, that will generate "\<subvolumes\>" in the output.